src/features/commands/stats.js

```diff
@@ -188,6 +195,9 @@ async function getGithubDownloads() {
       }
 
       obj[versionTag].downloads = downloadCount
+      const fullVersionString = version.published_at 
+      // Format: "created_at": "2024-04-22T01:48:05Z",
+      obj[versionTag].date = fullVersionString.substring(0, fullVersionString.indexOf("T"))
```

Comment:
I kinda dont like this hacky solution

Maybe do something like this:
```suggestion
      const date = new Date(version.published_at);
      const year = date.getFullYear();
      const month = String(date.getMonth() + 1).padStart(2, "0");
      const day = String(date.getDate()).padStart(2, "0");

      obj[versionTag].date= `${year}-${month}-${day}`;
```

Reply:
Does that actually work?

Reply:
Not tested, but should

Reply:
Is there any way to get it to spell out the month entirely?

Reply:
Fixed

Reply:
> Like "April 22, 2024"?

22 April 2024 but yes